src/main/groovy/liquibase/sdk/test/util/ChangeObjectTestHelper.groovy

```diff
@@ -31,7 +35,7 @@ class ChangeObjectTestHelper {
             def database = databaseUnderTest.database
             for (def changeLogEntry : TestUtils.getChangeLogPaths(databaseUnderTest,  TestConfig.instance.inputFormat).entrySet
                     ()) {
-                if (!changeObjects || (changeObjects && changeObjects.contains(changeLogEntry.key))) {
+                if (!changeObjectList || (changeObjectList && changeObjectList.contains(changeLogEntry.key))) {
                     inputList.add(TestInput.builder()
```

Comment:
@KushnirykOleh -- Doesn't groovy allow short-circuiting? i.e. if `changeObjectList` has already been evaluated true/false, then couldn't we drop the second `changeObjectList` from your conditional check? 
So write it as:
`if (!changeObjectList || changeObjectList.contains(changeLogEntry.key))` instead?